tests/jobs/misfit_preprocessor/test_misfit_preprocessor.py

```diff
@@ -160,18 +165,19 @@ def test_misfit_preprocessor_n_polynomials(num_polynomials):
     )
     measured_data = MockedMeasuredData(observations, simulated)
 
-    config = {}
+    config = {"clustering": {"method": method}, "scaling": {"threshold": 0.99}}
```

Comment:
Should we keep the spearman_correlation at the same threshold as default? And rather include it for the auto_scale only?

Reply:
I agree with the second comment, as to the first point, it does not really impact that test at all, because that job is only looking at the output of the clustering job, and the scaling has no impact on that. 

If you think it makes it more readable though, I can change it, but the reason I did not is to make it more precise what we are testing. But could be that I failed at that.

Reply:
I would say it makes it less readable, but we'll also not be changing as much. But get your first point and I suppose it's not an issue.